jacobian_follower/src/jacobian_follower/jacobian_follower.cpp

```diff
@@ -143,12 +141,12 @@ void JacobianFollower::debugLogState(const std::string prefix, robot_state::Robo
   ROS_DEBUG_STREAM_NAMED(LOGGER_NAME + ".joint_state", prefix << ss.str());
 }
 
-PlanResultMsg JacobianFollower::plan_return_msg(std::string const &group_name,
+PlanResultMsg JacobianFollower::plan(std::string const &group_name,
                                                 std::vector<std::string> const &tool_names,
                                                 std::vector<Eigen::Vector4d> const &preferred_tool_orientations,
                                                 std::vector<PointSequence> const &grippers,
-                                                double const max_velocity_scaling_factor,
-                                                double const max_acceleration_scaling_factor) {
+                                                double max_velocity_scaling_factor,
```

Comment:
you dropped the const.
nbd IMO, but `const` seems fine here

Reply:
added them back, good catch